cookbook/composables.md

```diff
@@ -17,27 +14,28 @@ export const useAuthStore = defineStore('auth', {
 })
 ```
 
-Keep in mind that **you can only return writable state** (e.g. a `ref()`). Here are some examples of composables that you can use:
+Ten en cuenta que **solo puedes devolver el estado de escritura** (por ejemplo, un `ref()`). Estos son algunos ejemplos de composables que puedes usar:
 
 - [useLocalStorage](https://vueuse.org/core/useLocalStorage/)
 - [useAsyncState](https://vueuse.org/core/useAsyncState/)
 
-Here are some examples of composables that cannot be used in an option stores (but can be used with setup stores):
+Estos son algunos ejemplos de composables que no se pueden usar en almacenes de opciones (pero se pueden usar con almacenes de configuración):
 
-- [useMediaControls](https://vueuse.org/core/useMediaControls/): exposes functions
-- [useMemoryInfo](https://vueuse.org/core/useMemory/): exposes readonly data
-- [useEyeDropper](https://vueuse.org/core/useEyeDropper/): exposes readonly data and functions
+- [useMediaControls](https://vueuse.org/core/useMediaControls/): expone funciones
+- [useMemoryInfo](https://vueuse.org/core/useMemory/): expone datos de solo lectura
+- [useEyeDropper](https://vueuse.org/core/useEyeDropper/): 
+expone datos y funciones de solo lectura
 
-## Setup Stores
+## Configuración de almacenes {#setup-stores}
```

Comment:
I believe that we should not translate thar as "Configuración de almacenes" because it seems that we are talking about a complete different thing. I think using "Almacenes de configuración" or "Almacenes con Setup" (this last one is because they use the same syntax as the setup function in vue components). What do you think?

Reply:
sure, but I have to change some translations in other files